manifests/ubuntu.pp

```diff
@@ -34,9 +34,9 @@
   }
 
   if !$skip_apt_key_trusting {
-    package { 'gnupg':
-      ensure => installed
-    }
+#    package { 'gnupg':
+#      ensure => installed
+#    }
```

Comment:
Unintended change?

Reply:
well yes and no. It should not have been in the PR but I needed to do that locally to avoid duplicate conflicts with the apt module.